src/main/resources/resources/logisim/strings/gui/gui.properties

```diff
@@ -642,9 +642,9 @@ argCircuitOption = Test the circuit of name passed as argument instead of "main"
 argClearOption = Clear application preferences at startup.
 argDuplicateSubstitutionError = Cannot substitute the same file multiple times.
 argEmptyOption = Use empty template.
-argGatesOption = Sets "shaped" or "rectangular" gate style.
+argGatesOption = Sets "ansi" or "iec" gate style.
```

Comment:
Since case should make no difference, can we replace `ansi` with `ANSI` and `iec` with `IEC` for better readability (multiple places)?

Reply:
Done